mbs_results/calculate_imputation_link.py

```diff
@@ -1,6 +1,8 @@
 import numpy as np
 import pandas as pd
 
+from mbs_results.flag_and_count_matched_pairs import count_matches
+
 
 def calculate_imputation_link(
```

Comment:
This is now a pretty long function, consider abstracting some of the new functionality into a lower level function

Reply:
Extracted some of the functionality into a new function